src/test/java/org/prebid/server/settings/mapper/JdbcStoredDataResultMapperTest.java

```diff
@@ -0,0 +1,129 @@
+package org.prebid.server.settings.mapper;
+
+import io.vertx.core.json.JsonArray;
+import io.vertx.ext.sql.ResultSet;
+import org.junit.Rule;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+import org.prebid.server.settings.model.StoredDataResult;
+
+import java.util.AbstractMap;
+import java.util.Arrays;
+import java.util.List;
+
+import static java.util.Collections.emptyList;
+import static java.util.Collections.singleton;
+import static java.util.Collections.singletonList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.BDDMockito.given;
+
+public class JdbcStoredDataResultMapperTest {
+
+    @Rule
+    public final MockitoRule mockitoRule = MockitoJUnit.rule();
+
+    @Mock
+    private ResultSet resultSet;
+
+    @Test
+    public void mapShouldReturnEmptyStoredResultWithErrorWhenResultSetHasEmptyResult() {
+        // given
+        given(resultSet.getResults()).willReturn(emptyList());
+
+        // when
+        final StoredDataResult result = JdbcStoredDataResultMapper.map(resultSet);
+
+        // then
+        assertThat(result.getStoredIdToRequest()).isEmpty();
+        assertThat(result.getStoredIdToImp()).isEmpty();
+        assertThat(result.getErrors()).hasSize(1)
+                .containsOnly("No stored requests or imps found");
+    }
+
+    @Test
+    public void mapShouldReturnEmptyStoredResultWithErrorWhenResultSetHasEmptyResultForGivenIDs() {
+        // given
+        given(resultSet.getResults()).willReturn(emptyList());
+
+        // when
+        final StoredDataResult result = JdbcStoredDataResultMapper.map(resultSet, singleton("reqId"), singleton("impId"));
```

Comment:
Pls apply code formatting, this line is slightly longer then allowed by checkstyle.